hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -705,27 +753,27 @@
         {
           bits: "2",
           name: "rx_d_i",
-          desc: "USB differential input."
+          desc: "USB data input from differential receiver, if available."
```

Comment:
just to be consistent should we say external differential receiver like you did in `usb_fs_rx`?